java/arcs/core/tools/Proto2Schema.kt

```diff
@@ -0,0 +1,138 @@
+package arcs.core.tools
+
+import arcs.core.data.FieldType
+import arcs.core.data.RecipeEnvelopeProto
+import arcs.core.data.Schema
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.file
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.PropertySpec
+import java.io.File
+
+class Proto2Schema : CliktCommand(
+    help = """Generates Schemas and Types from a protobuf-serialized manifest.
+    
+    This script reads schemas from a serialized manifest and generates Kotlin `Schema` and `Type` 
+    classes.""",
```

Comment:
Use `.trimIndent()` unless you intend the second paragraph to be indented by 4 spaces when output.

Reply:
Also, for readability:

```kotlin
    help = """
        Generates Schemas and Types from a protobuf-serialized manifest.

        This script reads schemas from a serialized manifest and generates Kotlin `Schema` and 
        `Type` implementations.
    """.trimIndent(),
```